CMakeLists.txt

```diff
@@ -946,6 +945,26 @@ else()
     message(FATAL_ERROR "${CMAKE_CXX_COMPILER_ID} not yet supported.")
 endif()
 
+if(NOT MSVC)
```

Comment:
Simply change lines 851 and 855. In 851 change -Wno-strict-aliasing to `-fno-strict-aliasing`. In 855 add it. There is no need for this whole new set of lines.

Reply:
You mean **851** (or 850 in my file), **889** (`basisu_transcoder.cpp` is the only one used in libktx_read), and for **clang** line **933** for the transcoder and for the **encoder** I'd still have to add **this same block** because for clang source file properties for `${BASISU_ENCODER_CXX_SRC}` is only set `if (${clang_version} VERSION_GREATER_EQUAL "14.0")`.

I'll do this change if you still want it (considering that clang also needs this so it's more than two lines that need to be adjusted), but will wait for your feedback.

Reply:
Are you getting warnings or errors when building with clang? Our CI is not showing any and it builds with clang on iOS, macOS, with Emscripten and via ClangCL on Windows.

Reply:
This is not about warnings or errors, but about disabling "strict aliasing" optimizations, which (according to comments in the basisu source files and [their CMakeLists.txt](https://github.com/BinomialLLC/basis_universal/blob/42c21d8066cd4b3f33e29272b61170e3b2e243bc/CMakeLists.txt#L1) could break their code.

Reply:
Keep this block but please remove the now unnecessary `-Wno-strict-aliasing' flag from line 850 as this renders it unnecessary